src/tests/functional/plugin/shared/include/behavior/compiled_model/device.hpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <gtest/gtest.h>
+
+#include <base/ov_behavior_test_utils.hpp>
```

Comment:
```suggestion
#include "base/ov_behavior_test_utils.hpp"
```
If this is OV include